### DISCUSSION TITLE
Proposal do downgrade iOS Platform

### DISCUSSION THREAD

**Discussion by lfoliveir4** [dooboolab/react-native-audio-recorder-player#347] [Q&A]
Helo! I'm currently trying to use your library and it's doing great. Congratulations!

However, your platform's target is currently the minimum for iOS 11. My React Native environment is fully updated to the latest stable version! (0.64.1) and yet the platform's iOS target is 10.0

Is it possible to downgrade the iOS version to 10 in order to continue using the library?

Problem a happens in the build for project execution!

![image](https://user-images.githubusercontent.com/32423942/123898041-ea781180-d93a-11eb-9517-a40cf0ec1155.png)

**Comment by juliorodriguespicpay** [comment]
I had the same problem!
Anyone has idea for handle with that problem?

**Reply by lfoliveir4** [reply]
@hyochan can you help us? The library is helping us a lot in many ways in a project!
